ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -36,6 +36,26 @@ class SignalSpec: QuickSpec {
 				expect(signal).to(beNil())
 			}
 
+			it("should not keep signal alive indefinitely even if it has an observer") {
+				weak var signal: Signal<AnyObject, NoError>? = {
+					let signal: Signal<AnyObject, NoError> = Signal.never
+					return signal
+				}()
+				expect(signal).to(beNil())
+			}
+
+			it("should not keep signal alive indefinitely even if it has an observer with retained disposable") {
```

Comment:
Can we rephrase this as "should deallocate the signal ..." instead of "should not keep the signal alive indefinitely ...". Same for the rest of the tests below.